.gitignore

```diff
@@ -1,4 +1,6 @@
-
+target/*
```

Comment:
This file is not required. This is part of local git file

Reply:
Removed the changes made in this file